tools/go-cli/build.gradle

```diff
@@ -37,6 +38,13 @@ ant.condition(property: "arch", value: "386") {
     os(arch: "x86_32")
 }
 
+// Sets the ext.dockerBuildArgs with the build build timestamp
+def setDockerBuildArgs() {
+    java.text.SimpleDateFormat sdf = new java.text.SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssZ")
+    String dateStr = sdf.format(new Date())
+    ext.dockerBuildArgs = "BUILDTIME="+dateStr
+}
```

Comment:
Could we change that function to actually `return` the values needed to pass to `ext.dockerBuildArgs` and assign them above? Makes the code easier to reason about.